packages/@expo/cli/src/start/server/middleware/InterstitialPageMiddleware.ts

```diff
@@ -49,8 +46,16 @@ export class InterstitialPageMiddleware extends ExpoMiddleware {
     let content = (await readFile(templatePath)).toString('utf-8');
 
     content = content.replace(/{{\s*AppName\s*}}/, appName);
-    content = content.replace(/{{\s*RuntimeVersion\s*}}/, runtimeVersion ?? '');
     content = content.replace(/{{\s*Path\s*}}/, this.projectRoot);
+    content = content.replace(/{{\s*Scheme\s*}}/, this.scheme ?? 'Unknown');
+
+    if (!runtimeVersion && sdkVersion) {
```

Comment:
I'd rather this be represented modularly like `projectVersion: { type: 'sdk' | 'runtime', version: string }`